### DISCUSSION TITLE
Allow option to also hide Username on individual entries

### DISCUSSION THREAD

**Discussion by xcpn** [keepassxreboot/keepassxc#10029] [Ideas]
While it's best to not use your Password Manager in front of other people, sometimes it's inevitable.

I think it would be great if we could also set specific entries to hide the usernames along with the passwords. Some websites can expose profile info just by the username, and having these entries only show the website name and not any credentials is a big deal.

**Comment by droidmonkey** [comment]
View-> Hide Usernames

**Reply by xcpn** [reply]
I'm aware of this setting, what I'm talking about is individual entries in the list. Not all Usernames globally.

**Reply by droidmonkey** [reply]
That doesn't make sense. If you are concerned about shoulder surfing, then you should be concerned about all your entries. Who is going to selectively mark entries as "more sensitive" than others? Plus that is a burden on the user interface and data model for zero gain.

**Reply by xcpn** [reply]
I believe there are use cases.

Some entries are more sensitive than other and having the option to hide them individually is great.

The option could be toggleable when you open an entry to edit, so no burden.
